hr_attendance_calendar/models/hr_attendance_rules.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2018 Compassion CH (http://www.compassion.ch)
+#    @author: Stephane Eicher <seicher@compassion.ch>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+from odoo import models, fields
+
+
+class HrAttendanceRules(models.Model):
+    _name = 'hr.attendance.rules'
+
+    ##########################################################################
+    #                                 FIELDS                                 #
+    ##########################################################################
+    name = fields.Char(string='Name')
+    threshold = fields.Float(string='Hours threshold in the day')
```

Comment:
Maybe it would better in the help parameter ? The name of the field should not be too long. I would take chance to give more explanation as I still don't really understand the meaning of these settings.